common/reviews/api/rush-lib.api.md

```diff
@@ -773,6 +773,7 @@ export interface _IPnpmOptionsJson extends IPackageManagerOptionsJsonBase {
     globalPackageExtensions?: Record<string, IPnpmPackageExtension>;
     globalPatchedDependencies?: Record<string, string>;
     globalPeerDependencyRules?: IPnpmPeerDependencyRules;
+    minimumReleaseAge?: number;
```

Comment:
Can you also add support for [`minimumReleaseAgeExclude`](https://pnpm.io/settings#minimumreleaseageexclude)?